spring-cloud-gcp-docs/src/main/asciidoc/storage.adoc

```diff
@@ -41,11 +41,17 @@ private Resource gcsResource;
 This creates a `Resource` object that can be used to read the object, among
 https://docs.spring.io/spring/docs/current/spring-framework-reference/html/resources.html#resources-resource[other possible operations].
 
-It is also possible to write to a `Resource`, although a cast to `WriteableResource` is required.
+It is also possible to write to a `Resource`, although a `WriteableResource` is required.
 
 [source,java]
 ----
-try (OutputStream os = ((WritableResource) gcsResource).getOutputStream()) {
+@Value("gs://[YOUR_GCS_BUCKET]/[GCS_FILE_NAME]")
+private WritableResource gcsResource;
```

Comment:
Wait, I thought this didn't work on `WritableResource`. See https://jira.spring.io/browse/SPR-10656.

Reply:
Well, correct. Have just tested against latest SF - doesn't work:
```
Caused by: java.lang.IllegalStateException: Cannot convert value of type 'java.lang.String' to required type 'org.springframework.core.io.WritableResource': no matching editors or conversion strategy found
```

I think I'll revert (or revise) this fix on merge.